### PR TITLE
BETA: Register 0x58b.is-a.dev

### PR DESCRIPTION
**Pull request by 0x58B1337**
Added `0x58b.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/0x58b.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "0x58B1337",
+        "email": "0xnullcode@gmail.com"
+    },
+    "record": {
+        "CNAME": "0x58b1337.github.io"
```
Comment (Stef-00012) [inline domains/0x58b.json:7]: you should complete your website

File: domains/0x58b1337.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "0x58B1337",
+        "email": "0xnullcode@gmail.com"
+    },
+    "record": {
+        "A": ["204.232.175.78"]
```
Comment (Stef-00012) [inline domains/0x58b1337.json:7]: I cannot reach your website, keeps on loading

File: domains/0x58.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "0x58B1337",
+        "email": "0xnullcode@gmail.com"
+    },
+    "record": {
+        "A": ["204.232.175.78"]
```
Comment (Stef-00012) [inline domains/0x58.json:7]: I cannot reach your website, keeps on loading

File: domains/lexy.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "0x58B1337",
+        "email": "0xnullcode@gmail.com"
+    },
+    "record": {
+        "A": ["204.232.175.78"]
```
Comment (Stef-00012) [inline domains/lexy.json:7]: I cannot reach your website, keeps on loading

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**